models/vista3d/docs/data_license.txt

```diff
@@ -0,0 +1,11 @@
+Third Party Licenses
+-----------------------------------------------------------------------
+
+/*********************************************************************/
+i. LUng Nodule Analysis 2016
```

Comment:
Hi @heyufan1995 @wyli , do you think we should use this data license? The Lung dataset seems not mentioned in the bundle